### ISSUE TITLE
Typed Text Is Not Visible

### ISSUE THREAD

**Issue by TimesEnemy** [signalapp/Signal-Android#7039]
I have:
- [x] searched open and closed issues for duplicates
- [x] read https://github.com/WhisperSystems/Signal-Android/wiki/Submitting-useful-bug-reports

----------------------------------------

### Bug description
When I go to type a message the text I type does not appear inside the input box.  The "spell-check" seems to detect the words as it makes recommendations, but the typed text is invisible.  This does not happen every time I try to send a message.  This happens occasionally, and seemingly random.

### Steps to reproduce
- I have not yet determined what spawns this, and as such do not currently know how to intentionally reproduce this
- When this happens, I "back" out of Signal, then go back into Signal (no cache clearing, or force stopping, et cetera), and Signal then shows typed text.

**Actual result:** The above description covers this.
**Expected result:** When typing a message, I should be able to see the text I am typing, character by character, prior to sending the message.

### Screenshots
Fail on my part.  I will grab some screenshots the next time this happens.

### Device info
**Device:** Google Pixel XL
**Android version:** 8.0.0
**Signal version:** 4.11.0

### Link to debug log
Yep, on a roll here.  I have not pulled a debug log yet, but will next time this happens.

**Comment by ByteHamster** [comment]
Notice: This is not a new behavior in 4.11.0. Have seen that happening for quite a while now.

**Comment by TimesEnemy** [comment]
ByteHamster,

I did not see any previously listed issues or mentions of this, but if you know of any please let me know.  If you have screenshots, or debug logs, that too would be helpful.

**Comment by johnhringiv** [comment]
I can confirm this bug and have the same setup as OP. Below is a debug log taken a few minutes after the issue occurred. I'll try to get a log and screenshot immediately after the next occurrence.

https://gist.github.com/anonymous/dba38a9439cc0af0c3c38f9fce15ada3

**Comment by TimesEnemy** [comment]
This just happened to me again.  I felt like an idiot though because it would not allow me to grab a screenshot ... doh.  Anyhow, I did take a picture of the phone, using another camera.  (I was not sure if I left the screen to adjust my screen capture settings if the issue would remain.)  Also, I snagged this log:

https://gist.github.com/anonymous/c9af308e28fbea4e27fd2ca360ba81e1

(The screenshot is not uploading to this forum.)
